test/modules/TwinTester/config/settings.json

```diff
@@ -8,5 +8,6 @@
     "serviceClientConnectionString": "",
     "StoragePath": "",
     "StorageOptimizeForPerformance": true,
-    "TwinTestMode": "TwinAllOperations"
+    "TwinTestMode": "TwinAllOperations",
+    "TestStartDelay":  "00:02:00"
```

Comment:
I suppose we already have the code to use the parameter if it exists?

Reply:
Yes. This will just set it by default